cmd/gpu-kubelet-plugin/main.go

```diff
@@ -104,11 +105,10 @@ func newApp() *cli.App {
 			EnvVars:     []string{"CONTAINER_DRIVER_ROOT"},
 		},
 		&cli.StringFlag{
-			Name:        "nvidia-ctk-path",
-			Value:       "/usr/bin/nvidia-ctk",
-			Usage:       "the path to use for the nvidia-ctk in the generated CDI specification. Note that this represents the path on the host.",
-			Destination: &flags.nvidiaCTKPath,
-			EnvVars:     []string{"NVIDIA_CTK_PATH"},
+			Name:        "nvidia-cdi-hook-path",
+			Usage:       "the path to use for the nvidia-cdi-hook in the generated CDI specification. Note that this represents the path on the host.",
+			Destination: &flags.nvidiaCDIHookPath,
+			EnvVars:     []string{"NVIDIA_CDI_HOOK_PATH"},
 		},
```

Comment:
As mentioned, it may be cleaner to remove this and readd it if we ever want to allow users to set it.

Reply:
I'm fine leaving it. I think it's cleaner keeping all of these under `flags` and I'd want/need to move it elsewhere if I didn't explicitly allow it being passed in as a flag.